src/hydrate.ts

```diff
@@ -67,8 +100,72 @@ const middleware = async <C extends Context>(
   await next();
 };
 
+const hydrateReplyFmt = async <C extends Context>(
```

Comment:
You can add these to the existing middleware. There is no compelling reasons to make 2 middlewares here.